crates/ruff_linter/src/rules/pyflakes/snapshots/ruff_linter__rules__pyflakes__tests__f401_import_submodules_in_function_scope.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: crates/ruff_linter/src/rules/pyflakes/mod.rs
+---
+F401 [*] `a` imported but unused
+ --> f401_preview_submodule.py:2:8
+  |
+2 | import a
```

Comment:
I found this a bit surprising but it's the same behavior as on main. 

Which makes me wonder if we should have used the assertion helper that compares preview with stable behavior.